scripts/publish_prod.sh

```diff
@@ -28,17 +28,39 @@ if [ -z "$AWS_SESSION_TOKEN" ]; then
     exit 1
 fi
 
+# Read the current version
+CURRENT_VERSION=$(node -pe "require('./package.json').version")
+
+# Read the desired version
+if [ -z "$1" ]; then
+    echo "Must specify a desired version number"
+    exit 1
+elif [[ ! $1 =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+    echo "Must use a semantic version, e.g., 3.1.4"
+    exit 1
+else
+    VERSION=$1
+fi
+
+# Confirm to proceed
+read -p "About to bump the version from ${CURRENT_VERSION} to ${VERSION}, and publish. Continue (y/n)?" CONT
```

Comment:
Is it worth thinking about how to add a `-f` flag so that this can be used by SDP for automatic releases, or should we worry about that in Q3?

Reply:
I think we worry about that later, we may need to replace this script with something completely different to configure the release process.

Reply:
👍